migrations/20200617090839_add_community_privacy_settings.js

```diff
@@ -0,0 +1,24 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.table('communities', table => {
+    table.boolean('is_public').defaultTo(false)
+  }).then(() => knex.schema.table('communities', table => {
+    table.boolean('is_auto_joinable').defaultTo(false)
+  })).then(() => knex.schema.table('communities', table => {
+    table.boolean('public_member_directory').defaultTo(false)
+  })).then(() => knex.raw(`CREATE INDEX public_communities_idx ON communities USING btree (is_public);`))
```

Comment:
been reading up on indexes, i think we might want a single index that includes all three of these columns? but obviously not that important right now

Reply:
ok done 👍